gateway/webhook/webhook.go

```diff
@@ -247,6 +248,22 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 				continue
 			}
 
+			if misc.ContainsString(sourceListForParsingParams, strings.ToLower(breq.sourceType)) {
```

Comment:
does it make sense to convert both of them to lower case for comparison check?

Reply:
can do that